service/create/service.go

```diff
@@ -824,38 +824,19 @@ func (s *Service) onAdd(obj interface{}) {
 		return
 	}
 
-	// Run workers
-	anyWorkersCreated, workerIDs, err := s.runMachines(runMachinesInput{
-		clients:             clients,
-		cluster:             cluster,
-		tlsAssets:           tlsAssets,
-		bucket:              bucket,
-		securityGroup:       workersSecurityGroup,
-		subnet:              publicSubnet,
-		clusterName:         cluster.Name,
-		keyPairName:         cluster.Name,
-		instanceProfileName: policy.GetName(),
-		prefix:              prefixWorker,
-	})
-	if err != nil {
-		s.logger.Log("error", errgo.Details(err))
-		return
-	}
-
 	// If the policy couldn't be created and some instances didn't exist before, that means that the cluster
 	// is inconsistent and most problably its deployment broke in the middle during the previous run of
 	// aws-operator.
-	if (anyMastersCreated || anyWorkersCreated) && (kmsKeyErr != nil || policyErr != nil) {
+	if anyMastersCreated && (kmsKeyErr != nil || policyErr != nil) {
```

Comment:
@nhlfr do you know if this still works as intended? i.e. now that we don't use `runMachines` for the workers, do we still catch inconsistent states?